packages/stark-ui/src/modules/app-footer/components/app-footer.component.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*tslint:disable:completed-docs*/
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import {async, ComponentFixture, inject, TestBed} from "@angular/core/testing";
+import {
+	STARK_LOGGING_SERVICE
+} from "@nationalbankbelgium/stark-core";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { StarkAppFooterComponent } from "./app-footer.component";
+import { StarkSvgViewBoxModule } from "../../svg-view-box/svg-view-box.module";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { MatButtonModule } from "@angular/material/button";
+import { TranslateModule, TranslateService } from "@ngx-translate/core";
+
+describe("AppLogoutComponent", () => {
+	let component: StarkAppFooterComponent;
+	let fixture: ComponentFixture<StarkAppFooterComponent>;
+	
+	const currentYear: string =  new Date().getFullYear().toString();
+
+	let translateService: TranslateService;
+	/**
+	 * async beforeEach
+	 */
+	beforeEach(async(() => {
+		return (
+			TestBed.configureTestingModule({
+				imports: [MatTooltipModule, MatButtonModule, StarkSvgViewBoxModule, TranslateModule.forRoot()],
```

Comment:
I think these modules are not used: `MatTooltipModule`, `MatButtonModule`, `StarkSvgViewBoxModule` ;)